diff -uNr e-smith-radiusd-1.0.0.orig/root/etc/e-smith/templates/etc/raddb/radiusd.conf/65authorization e-smith-radiusd-1.0.0/root/etc/e-smith/templates/etc/raddb/radiusd.conf/65authorization
--- e-smith-radiusd-1.0.0.orig/root/etc/e-smith/templates/etc/raddb/radiusd.conf/65authorization	2005-06-11 20:52:15.000000000 +0200
+++ e-smith-radiusd-1.0.0/root/etc/e-smith/templates/etc/raddb/radiusd.conf/65authorization	1970-01-01 01:00:00.000000000 +0100
@@ -1,50 +0,0 @@
-{
-#  Authorization. First preprocess (hints and huntgroups files),
-#  then realms, and finally look in the "users" file.
-#
-#  The order of the realm modules will determine the order that
-#  we try to find a matching realm.
-#
-#  Make *sure* that 'preprocess' comes before any realm if you 
-#  need to setup hints for the remote radius server
-}
-authorize \{
-{
-	#  The preprocess module takes care of sanitizing some bizarre
-	#  attributes in the request, and turning them into attributes
-	#  which are more standard.
-	#
-	#  It takes care of processing the 'raddb/hints' and the
-	#  'raddb/huntgroups' files.
-	#
-	#  It also adds the %\{Client-IP-Address\} attribute to the request.
-}	preprocess
-{	
-	#  If you are using multiple kinds of realms, you probably
-	#  want to set "ignore_null = yes" for all of them.
-	#  Otherwise, when the first style of realm doesn't match,
-	#  the other styles won't be checked.
-}	suffix
-	ntdomain
-{	
-	#  This module takes care of EAP-PEAP authentication.
-	#
-	#  It also sets the EAP-Type attribute in the request
-	#  attribute list to the EAP type from the packet.
-}	eap
-{
-	#  If the users are logging in with an MS-CHAP-Challenge
-	#  attribute for authentication, the mschap module will find
-	#  the MS-CHAP-Challenge attribute, and add 'Auth-Type := MS-CHAP'
-	#  to the request, which will cause the server to then use
-	#  the mschap module for authentication.
-}	mschap
-{	
-	#  If you are using /etc/smbpasswd, and are also doing
-	#  mschap authentication, the un-comment this line, and
-	#  configure the 'smbpasswd' module, above.
-}	smbpasswd
-{	
-	#  Read the 'users' file
-}	files
-\}
diff -uNr e-smith-radiusd-1.0.0.orig/root/etc/e-smith/templates/etc/raddb/radiusd.conf/65authorization00init e-smith-radiusd-1.0.0/root/etc/e-smith/templates/etc/raddb/radiusd.conf/65authorization00init
--- e-smith-radiusd-1.0.0.orig/root/etc/e-smith/templates/etc/raddb/radiusd.conf/65authorization00init	1970-01-01 01:00:00.000000000 +0100
+++ e-smith-radiusd-1.0.0/root/etc/e-smith/templates/etc/raddb/radiusd.conf/65authorization00init	2007-02-22 09:19:43.000000000 +0100
@@ -0,0 +1,11 @@
+{
+#  Authorization. First preprocess (hints and huntgroups files),
+#  then realms, and finally look in the "users" file.
+#
+#  The order of the realm modules will determine the order that
+#  we try to find a matching realm.
+#
+#  Make *sure* that 'preprocess' comes before any realm if you 
+#  need to setup hints for the remote radius server
+}
+authorize \{
diff -uNr e-smith-radiusd-1.0.0.orig/root/etc/e-smith/templates/etc/raddb/radiusd.conf/65authorization40default e-smith-radiusd-1.0.0/root/etc/e-smith/templates/etc/raddb/radiusd.conf/65authorization40default
--- e-smith-radiusd-1.0.0.orig/root/etc/e-smith/templates/etc/raddb/radiusd.conf/65authorization40default	1970-01-01 01:00:00.000000000 +0100
+++ e-smith-radiusd-1.0.0/root/etc/e-smith/templates/etc/raddb/radiusd.conf/65authorization40default	2007-02-22 09:19:58.000000000 +0100
@@ -0,0 +1,38 @@
+{
+	#  The preprocess module takes care of sanitizing some bizarre
+	#  attributes in the request, and turning them into attributes
+	#  which are more standard.
+	#
+	#  It takes care of processing the 'raddb/hints' and the
+	#  'raddb/huntgroups' files.
+	#
+	#  It also adds the %\{Client-IP-Address\} attribute to the request.
+}	preprocess
+{	
+	#  If you are using multiple kinds of realms, you probably
+	#  want to set "ignore_null = yes" for all of them.
+	#  Otherwise, when the first style of realm doesn't match,
+	#  the other styles won't be checked.
+}	suffix
+	ntdomain
+{	
+	#  This module takes care of EAP-PEAP authentication.
+	#
+	#  It also sets the EAP-Type attribute in the request
+	#  attribute list to the EAP type from the packet.
+}	eap
+{
+	#  If the users are logging in with an MS-CHAP-Challenge
+	#  attribute for authentication, the mschap module will find
+	#  the MS-CHAP-Challenge attribute, and add 'Auth-Type := MS-CHAP'
+	#  to the request, which will cause the server to then use
+	#  the mschap module for authentication.
+}	mschap
+{	
+	#  If you are using /etc/smbpasswd, and are also doing
+	#  mschap authentication, the un-comment this line, and
+	#  configure the 'smbpasswd' module, above.
+}	smbpasswd
+{	
+	#  Read the 'users' file
+}	files
diff -uNr e-smith-radiusd-1.0.0.orig/root/etc/e-smith/templates/etc/raddb/radiusd.conf/65authorization99end e-smith-radiusd-1.0.0/root/etc/e-smith/templates/etc/raddb/radiusd.conf/65authorization99end
--- e-smith-radiusd-1.0.0.orig/root/etc/e-smith/templates/etc/raddb/radiusd.conf/65authorization99end	1970-01-01 01:00:00.000000000 +0100
+++ e-smith-radiusd-1.0.0/root/etc/e-smith/templates/etc/raddb/radiusd.conf/65authorization99end	2007-02-22 09:20:09.000000000 +0100
@@ -0,0 +1 @@
+\}
diff -uNr e-smith-radiusd-1.0.0.orig/root/etc/e-smith/templates/etc/raddb/radiusd.conf/80accounting e-smith-radiusd-1.0.0/root/etc/e-smith/templates/etc/raddb/radiusd.conf/80accounting
--- e-smith-radiusd-1.0.0.orig/root/etc/e-smith/templates/etc/raddb/radiusd.conf/80accounting	2007-02-22 10:11:12.000000000 +0100
+++ e-smith-radiusd-1.0.0/root/etc/e-smith/templates/etc/raddb/radiusd.conf/80accounting	1970-01-01 01:00:00.000000000 +0100
@@ -1,11 +0,0 @@
-{
-#
-#  Accounting.  Log the accounting data.
-#
-}accounting \{
-{	#
-	#  Create a 'detail'ed log of the packets.
-	#  Note that accounting requests which are proxied
-	#  are also logged in the detail file.
-}	detail
-\}
diff -uNr e-smith-radiusd-1.0.0.orig/root/etc/e-smith/templates/etc/raddb/radiusd.conf/80accounting00init e-smith-radiusd-1.0.0/root/etc/e-smith/templates/etc/raddb/radiusd.conf/80accounting00init
--- e-smith-radiusd-1.0.0.orig/root/etc/e-smith/templates/etc/raddb/radiusd.conf/80accounting00init	1970-01-01 01:00:00.000000000 +0100
+++ e-smith-radiusd-1.0.0/root/etc/e-smith/templates/etc/raddb/radiusd.conf/80accounting00init	2007-02-22 09:21:13.000000000 +0100
@@ -0,0 +1,5 @@
+{
+#
+#  Accounting.  Log the accounting data.
+#
+}accounting \{
diff -uNr e-smith-radiusd-1.0.0.orig/root/etc/e-smith/templates/etc/raddb/radiusd.conf/80accounting40default e-smith-radiusd-1.0.0/root/etc/e-smith/templates/etc/raddb/radiusd.conf/80accounting40default
--- e-smith-radiusd-1.0.0.orig/root/etc/e-smith/templates/etc/raddb/radiusd.conf/80accounting40default	1970-01-01 01:00:00.000000000 +0100
+++ e-smith-radiusd-1.0.0/root/etc/e-smith/templates/etc/raddb/radiusd.conf/80accounting40default	2007-02-22 09:21:29.000000000 +0100
@@ -0,0 +1,5 @@
+{	#
+	#  Create a 'detail'ed log of the packets.
+	#  Note that accounting requests which are proxied
+	#  are also logged in the detail file.
+}	detail
diff -uNr e-smith-radiusd-1.0.0.orig/root/etc/e-smith/templates/etc/raddb/radiusd.conf/80accounting99end e-smith-radiusd-1.0.0/root/etc/e-smith/templates/etc/raddb/radiusd.conf/80accounting99end
--- e-smith-radiusd-1.0.0.orig/root/etc/e-smith/templates/etc/raddb/radiusd.conf/80accounting99end	1970-01-01 01:00:00.000000000 +0100
+++ e-smith-radiusd-1.0.0/root/etc/e-smith/templates/etc/raddb/radiusd.conf/80accounting99end	2007-02-22 09:21:36.000000000 +0100
@@ -0,0 +1 @@
+\}
